### DISCUSSION TITLE
Can't resolve 'next/headers'

### DISCUSSION THREAD

**Discussion by revmischa** [nextauthjs/next-auth#5764] [Help]
### Environment

Using local `main` checkout linked, next 12.3, ESM

### Reproduction URL

https://github.com/jetbridge/cdk-nextjs

### Describe the issue

../../next-auth/packages/next-auth/next/index.js
Module not found: Can't resolve 'next/headers' in '/home/cyber/dev/next-auth/packages/next-auth/next'

Import trace for requested module:
../../next-auth/packages/next-auth/next/index.js
../../next-auth/packages/next-auth/index.js

### How to reproduce

Try to build next in standalone mode with latest main (58e30a6af6e8b1f9084f5edc83aa83ddf44653ec)

### Expected behavior

Dunno

**Comment by biplobsd** [comment]
For quick solved:
Downgrade next-auth to 
`npm i next-auth@4.15.0`

**Comment by ThangHuuVu** [comment]
Moving this to a discussion because it's not an issue with the library but more about how to start up the dev app internally
